Makefile

```diff
@@ -87,6 +87,13 @@ mypy: regen-metaparser
 black:
 	black pegen tatsu test scripts
 
+format-pegen:
+	$(eval COMPILE_OPTIONS = $(shell python-config --cflags))
```

Comment:
Is $(eval) a shell or a Make primitive here? If shell, does the variable get passed to the next command? I thought make forks a new shell for each line?

Reply:
> Is $(eval) a shell or a Make primitive here?

Is a makefile primitive (https://www.gnu.org/software/make/manual/html_node/Eval-Function.html) unless I am misunderstanding something here.